src/components/UserInfoPopover.js

```diff
@@ -83,13 +83,21 @@ export default function UserInfoPopover(props) {
                 <div id="user-balance-usd">
                   {`${props.userInfo.balanceInUSDT} USDT`}
                 </div>
+                <div className="user-info-row show-withdraw-button">
+                  <Button startIcon={<OpenInNewIcon />} onClick={props.promptForWithdraw}>Withdraw</Button>
+                </div>
                 {/* <div id="user-balance-usd">
                   <Chip label="$ 56" color="primary"/>
                 </div> */}
           </div>
           <div className="user-info-row">
             <div className="user-info-deposit-label">Deposit funds to below address</div>
             <div className="user-info-deposit-address">{props.userInfo.userAddress}</div>
+            <div className="user-info-deposit-info">
+              <div className="tooltipLabel">why this address is different than my portis address?
+                <div className="tooltiptext">Tooltip text Tooltip text Tooltip textTooltip text Tooltip text Tooltip text Tooltip textTooltip text Tooltip text Tooltip text Tooltip textTooltip text</div>
```

Comment:
Text should be. 
This address is your wallet address on blockchain whose owner is your portis account address. Funds can be withdrawn only with your permission from this wallet. This on-chain wallet allows us to do gasless transactions for you so you do not have to pay the gas fees for your transactions.